shared/src/main/kotlin/org/jetbrains/kotlin/konan/target/ClangArgs.kt

```diff
@@ -61,22 +67,22 @@ class ClangArgs(private val configurables: Configurables) : Configurables by con
                     listOf("-target", targetArg!!, "--sysroot=$absoluteTargetSysRoot", "-Xclang", "-flto-visibility-public-std")
 
                 KonanTarget.MACOS_X64 ->
-                    listOf("--sysroot=$absoluteTargetSysRoot", "-mmacosx-version-min=10.11")
+                    listOf("--sysroot=$absoluteTargetSysRoot", "-mmacosx-version-min=$osVersionMin")
 
                 KonanTarget.IOS_ARM32 ->
-                    listOf("-stdlib=libc++", "-arch", "armv7", "-isysroot", absoluteTargetSysRoot, "-miphoneos-version-min=9.0.0")
+                    listOf("-stdlib=libc++", "-arch", "armv7", "-isysroot", absoluteTargetSysRoot, "-miphoneos-version-min=$osVersionMin")
```

Comment:
Isn't `osVersionMin` actually differ from `"9.0.0"`?

Reply:
Well, it's 9.0. What kind of problem may it cause?